src/hooks/content/useContent.js

```diff
@@ -0,0 +1,22 @@
+import { useSelector, shallowEqual } from 'react-redux';
+
+export function useContent() {
+  const data = useSelector((state) => state.content.data, shallowEqual);
```

Comment:
@tiberiuichim what do you think of this?

Reply:
@sneridagh I'm not sure, in principle is fine, but I have some concerns. 

I'm genuinely curious about all the "ceremony", with multiple hook calls and shallowEqual. Why not just return state.content?

```
const content = useSelector((state) => state.content);
return {
  data: content.data,
  loading: content.get.loading,
  ...
}
```

I think that at this point, while we wait for useQuery, the hook is underpowered. It can't be used for content subrequests, which highlights the other problem: the hook right now only takes care of yielding content info, it doesn't trigger a `getContent` action.

Following this train of thought, I think it's better to either go full useQuery or not at all. If we introduce this, we'll have multiple variations of this type of code in the codebase (old style, useContent, etc), which will all have to be migrated to the new variation of `useContent`.

The counter-argument is that the `useContent` with subrequests is not needed in vanilla Volto, so it's fine to introduce it right now as it is and enhance it with params and options later.

Reply:
Thanks for the review! Let's try to talk today!